paasta_tools/utils.py

```diff
@@ -2734,6 +2734,15 @@ def get_tron_k8s_cluster_overrides(self) -> Dict[str, str]:
         """
         return self.config_dict.get("tron_k8s_cluster_overrides", {})
 
+    def get_uwsgi_offset_multiplier(self) -> float:
+        """
+        Temporary configuration to allow us to slowly deprecate the usage of `offset` in uwsgi-based autoscaling
+        configurations without making a single massive change to how usage is calculated.
+
+        To be removed once PAASTA-17840 is done.
+        """
+        return self.config_dict.get("uwsgi_offset_multiplier", 1.0)
```

Comment:
probably need to add this attribute to `SystemPaastaConfigDict`

Reply:
yup - just did that after running mypy locally (and between you loading this PR and reviewing it :p)